snapshots/requests.http

```diff
@@ -0,0 +1,16 @@
+### Create a new Entity with initial id of "entity-id" and name of "Entity Name".
```

Comment:
Would it make sense to point to these in the readme, to share how others can test this themselves?
Added on that, the `*.http` files are awesome, but they do require Intellij. So if you would draft up some description in the readme, be sure to base your self on something like `curl` or `httpie`.

Reply:
Added curl examples to the README